src/main/java/com/eerussianguy/blazemap/feature/maps/WorldMapGui.java

```diff
@@ -171,7 +182,7 @@ protected void init() {
     private void updateLegend() {
         var legend = WrappedComponent.ofNullable(map.getMapType().getLayers().iterator().next().value().getLegendWidget());
 
-        if(legend == null) {
+        if (legend == null) {
```

Comment:
I just noticed all these.... we can't have these changed every time you touch a file.
Can you undo all these spaces and configure your formatter to be more in line with the style of this project please?
I'm not going to open a comment for every instance, I picked this one just so it's clear what I'm referring to. #